extensions/amp-date-picker/0.1/test/validator-amp-date-picker.html

```diff
@@ -142,5 +148,11 @@
   <!-- Invalid: single amp-date-picker with minimum nights -->
   <amp-date-picker layout="fixed-height" height="360" minimum-nights="0">
   </amp-date-picker>
+  <!-- Invalid: single amp-date-picker with start-date attribute -->
+  <amp-date-picker layout="fixed-height" height="360" start-date="2018-01-01">
+  </amp-date-picker>
+  <!-- Invalid: range amp-date-picker with date attribute -->
```

Comment:
This one didn't result in it being invalid. Is this supposed to be range with end-date attribute?

Reply:
Ah good catch. I neglected to put the `type="range"` on the tag. Thanks!

Reply:
https://github.com/ampproject/amphtml/pull/18354